transformer_engine/pytorch/transformer.py

```diff
@@ -77,6 +77,48 @@ def forward(self, hidden_state: torch.Tensor) -> torch.Tensor:
         output = hidden_state.div(keep_prob) * random_tensor
         return output
 
+class _SplitLastDim(torch.autograd.Function):
+    """"""
+
+    @staticmethod
+    def forward(ctx,
+                mixed_x_layer: torch.Tensor,
+                num_parts: int
+    ) -> Tuple[torch.Tensor, ...]:
+        return split_tensor_along_dim(mixed_x_layer, -1, num_parts)
+
+    @staticmethod
+    def backward(ctx,
+                 *args):
+        assert len(args) > 0
```

Comment:
Error message would be nice along with assert

Reply:
Ok, will change that,